test/KalmanTest.cpp

```diff
@@ -87,55 +76,55 @@ namespace rtt {
         ASSERT_DOUBLE_EQ(C(1, 1), 1);
     }
 
-
-    TEST(KalmanTest, ZXK) {
-
-        kalmanFilter ZXKtest;
-
-        ZXKtest.setZ(1, 1.0, 2.0, 3.0, 1.0);
-        for (int j = 0; j < 10000; ++j) {
-            ZXKtest.kalmanUpdate();
-        }
-
-        float testK1 = ZXKtest.getK(1);
-
-        Position testX = ZXKtest.getPos(1);
-        EXPECT_FLOAT_EQ(testX.x, 1);
-        EXPECT_FLOAT_EQ(testX.y, 2);
-        EXPECT_FLOAT_EQ(testX.rot, 3);
-
-        ZXKtest.setZ(1, 2.0, 4.0, 6.0, 2.0);
-        for (int j = 0; j < 10000; ++j) {
-            ZXKtest.kalmanUpdate();
-        }
-
-        testX = ZXKtest.getPos(1);
-        EXPECT_FLOAT_EQ(testX.x, 2);
-        EXPECT_FLOAT_EQ(testX.y, 4);
-        EXPECT_FLOAT_EQ(testX.rot, 6);
-
-        float testK2 = ZXKtest.getK(1);
-        EXPECT_FLOAT_EQ(testK1, testK2);
-
-    }
-
-    TEST(KalmanTest, frame){
-
-        kalmanFilter frametest;
-
-        auto * frame = new DetectionFrame();
-        kalman_dummy_frame(1.0, 1.0, 2.0, 2.0, 0.0, frame);
-
-        frametest.newFrame(*frame);
-
-        for (int j = 0; j < 10000; ++j) {
-            frametest.kalmanUpdate();
-        }
-
-        Position testX = frametest.getPos(0);
-        EXPECT_FLOAT_EQ(testX.x, 2);
-        EXPECT_FLOAT_EQ(testX.y, 2);
-        EXPECT_FLOAT_EQ(testX.rot, 0);
-    }
+//
+//    TEST(KalmanTest, ZXK) {
```

Comment:
Were these tests failing? In any case I don't like to see commented code.

Reply:
Fully agree, because we are using Git there is no need to have commented code.